services/merak-network/activities/services_activity.go

```diff
@@ -44,6 +45,11 @@ func DoServices(services []*pb.InternalServiceInfo) (string, error) {
 					log.Printf("numberOfService %s", numberOfService)
 				}
 			}
+			if service.Cmd == "alcorIp" {
+				utils.ALCORURL = service.Url
+				log.Printf("idServiceMap[currentId].Url %s", service.Url)
+				log.Printf("utils.ALCORURL %s", utils.ALCORURL)
+			}
```

Comment:
@yanmo96 Did you handle the error if `service.Url` is empty here?

Reply:
I will put a return here. But if `service.Cmd` or `service.Url` is empty or did not match or wrong. The later on register node or create network part will return the error, with the url showing there is noting in it, or the url is wrong.